<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FillInformationWindow</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>835</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1067</width>
    <height>737</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1202</width>
    <height>835</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>信息填写</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="707,412">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTableWidget" name="turbine_table">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>707</width>
           <height>757</height>
          </size>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <column>
          <property name="text">
           <string>机组编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>机型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>叶片长度</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>轮毂高度</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>412</width>
             <height>444</height>
            </size>
           </property>
           <property name="title">
            <string>添加机组及信息</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>29</y>
              <width>391</width>
              <height>351</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="Label">
                 <property name="text">
                  <string>      电    厂：</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QTextBrowser" name="windfarm_text">
                 <property name="maximumSize">
                  <size>
                   <width>220</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="Label_2">
                 <property name="text">
                  <string>      机组编号：</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="machine_le">
                 <property name="maximumSize">
                  <size>
                   <width>220</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="Label_3">
                 <property name="text">
                  <string>      机    型：</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="turbineType_combo">
                 <property name="maximumSize">
                  <size>
                   <width>220</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="Label_4">
                 <property name="text">
                  <string>      叶片长度：</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="bladeLength">
                 <property name="maximumSize">
                  <size>
                   <width>220</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="Label_5">
                 <property name="text">
                  <string>      轮毂高度：</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="wheelsHeight">
                 <property name="maximumSize">
                  <size>
                   <width>220</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="addTurbine_btn">
                 <property name="minimumSize">
                  <size>
                   <width>245</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>245</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>添加机组到本电厂</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>412</width>
             <height>187</height>
            </size>
           </property>
           <property name="title">
            <string>查找或修改机组</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>19</y>
              <width>391</width>
              <height>161</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>45</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="searchTurbine_btn">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>查找机组</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QPushButton" name="deleteTurbine_btn">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>删除机组</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="changeTurbineInfo_btn">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>更改机组信息</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>6</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>67</height>
            </size>
           </property>
           <property name="text">
            <string>信息填写完成并保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
